app/components/admin/schools/cohort_component.html.erb

```diff
@@ -9,16 +9,31 @@
 
   <% elsif fip? %>
 
-    <% partnership_components.each do |pc| %>
-      <%= pc %>
-    <% end %>
+    <% if has_partnerships_or_relationships? %>
+      <% partnership_components.each do |pc| %>
+        <%= pc %>
+      <% end %>
```

Comment:
Would it be helpful to always show these Training provider and Delivery partner rows whenever the training programme is the full induction programme, even if the partnerships haven’t been set up?  (They could show "Not set up yet" or similar in the value column?)

Reply:
I'm not sure what the actual logic needs to be here but each partnership component will be its own summary list (card).

I'm happy to tweak it but would rather see this update released first as it fixes a bug.

Reply:
I guess I mean that if there's no partnership set up for a cohort (but there should be, as they've picked the full induction programme), then it might be worth showing that explicitly?

But agree that this can get released first!